src/presentational-components/shared/table-toolbar-view.js

```diff
@@ -207,6 +207,7 @@ TableToolbarView.propTypes = {
   rowWrapper: propTypes.any,
   isCompact: propTypes.bool,
   borders: propTypes.bool,
+  checkedRows: propTypes.array,
```

Comment:
This should be an `arrayOf` prop type.

Reply:
When you say arrayOf for this and selectedPermissions, do you mean for me to give it a shape as an argument? since: propTypes.arrayOf(expectedType),